components/QueryControls/index.tsx

```diff
@@ -194,7 +286,7 @@ export default function QueryControls({
           />
           {allowedTaxonomies.map((taxonomy) => (
             <Fragment key={taxonomy.slug}>
-              { /* @ts-ignore */ }
+              { /* @ts-ignore */}
```

Comment:
another strange spacing change. at least let's do this for symmetry:

```suggestion
              {/* @ts-ignore */}
```